From yoichi_yuasa@tripeaks.co.jp Wed Aug  2 17:55:00 2006
Date: Thu, 3 Aug 2006 09:54:24 +0900
From: Yoichi Yuasa <yoichi_yuasa@tripeaks.co.jp>
To: gregkh@suse.de
Cc: linux-usb-devel@lists.sourceforge.net
Subject: USB: removed a unbalanced #endif from ohci-au1xxx.c
Message-Id: <20060803095424.5d2fe888.yoichi_yuasa@tripeaks.co.jp>

This patch has removed a unbalanced #endif from ohci-au1xxx.c .
Please apply before 2.6.18 release.

Error message was:
In file included from drivers/usb/host/ohci-hcd.c:909:
drivers/usb/host/ohci-au1xxx.c:113:2: #endif without #if

Signed-off-by: Yoichi Yuasa <yoichi_yuasa@tripeaks.co.jp>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-au1xxx.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/host/ohci-au1xxx.c
+++ gregkh-2.6/drivers/usb/host/ohci-au1xxx.c
@@ -110,7 +110,6 @@ static void au1xxx_start_ohc(struct plat
 
 	printk(KERN_DEBUG __FILE__
 	": Clock to USB host has been enabled \n");
-#endif
 }
 
 static void au1xxx_stop_ohc(struct platform_device *dev)
